lib/minimagick/tool.rb

```diff
@@ -0,0 +1,16 @@
+# Hack to monkey-patch MiniMagick to always add the 'quiet'
+# option to every imagemagick command line.
+class MiniMagick::Tool
+  class_attribute :quiet_arg
+  self.quiet_arg = false
+
+  prepend(Module.new do
+    def command
+      if quiet_arg
+        [*executable, *(['-quiet'] + args)]
+      else
+        super
+      end
+    end
+  end)
+end
```

Comment:
I think the safe way is

a) make our file at a path that isn't going ot conflict with any "real" file

b) make sure our file refers to Minimagick::Tool first in some way other than just `class MiniMagick::Tool`, some way that will _raise_ if there _is_ no Minimagick::Tool already previously, rather than just defining it to be only what's in this file.